tests/zlib/ref.txt

```diff
@@ -1,4 +1,4 @@
-zlib version 1.2.5 = 4688, compile flags = 85
+zlib version 1.2.5 = 4688, compile flags = 149
```

Comment:
what's this?

Reply:
zlib apparently prints out some flags and one of them reflects the size of `off_t`.

https://github.com/kripken/emscripten/blob/incoming/tests/zlib/zutil.c#L55
https://github.com/kripken/emscripten/blob/incoming/tests/zlib/example.c#L531